.pre-commit-config.yaml

```diff
@@ -32,7 +32,7 @@ repos:
           - flake8-bugbear
           - flake8-comprehensions
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: 'v1.18.2'
+    rev: 'v2.7.1'
```

Comment:
@PeteCoward any chance you could drop the linting dependencies from package.json while at this? This way we keep it just in pre-commit.

https://github.com/wagtail/cookiecutter-wagtail-package/pull/47/files has an example.

If not, will make that a follow-up task

Reply:
I had a go.

I snuck in a change from `pip install -e .[testing] -U` to `pip install -e '.[testing]' -U` too, which caught me out following the README.

I did not make the change to the pre-commit run all files README comment, as I was not sure of the rationale behind it.